pkg/buildpack/builder.go

```diff
@@ -268,15 +268,23 @@ func validateBuildpacks(mainBP Buildpack, depBPs []Buildpack) error {
 		bpd := bp.Descriptor()
 		for _, orderEntry := range bpd.Order {
 			for _, groupEntry := range orderEntry.Group {
-				if _, ok := depsWithRefs[groupEntry.BuildpackInfo.FullName()]; !ok {
+				bpFullName, err := groupEntry.BuildpackInfo.FullNameWithVersion()
+				if err != nil {
```

Comment:
It's a little strange that we're not using the `err` being returned from this method.

Could we figure out a way to incorporate the error message from the method into the message we are returning?

This would most likely change the exact verbiage we wanted but it would hopefully lead to a more detailed error message.

Reply:
> It's a little strange that we're not using the err being returned from this method.

Completely agree, the idea was to get the error message that you [suggested](https://github.com/buildpacks/pack/pull/1437#discussion_r868451760).
I'll update this and find a way to use the error as well as keep the error message relevant.